db/migrate/20230510075327_create_post.rb

```diff
@@ -11,12 +11,3 @@ def change
     end
```

Comment:
![image](https://github.com/otmanTR/BlogAppRoR/assets/93039359/e8ba5783-273c-4899-904b-5f1132e1b65f)

- [ ] It would be nice if you fix your live demo link to make your readme descriptive and professional :blush: If there is not live demo section then it would be nice if you remove this link :+1:

Reply:
Thank you